api/admin.py

```diff
@@ -175,3 +175,49 @@ def get_messages(message_filter: str = 'all'):
         code = OK
 
     return response, code
+
+# Route to get a singular message with given id.
+@admin.route('/get-message/<message_id>', methods=['GET'])
+def get_message(message_id: int):
+    """Get a singular message with the given id
+    }
```

Comment:
I am assuming you know the form of the message being returned, but if you could add the form here in the comments just for documentation that would be great

Reply:
See under the 'get listing' routes to see what I mean; JSON format

Reply:
I agree